packages/main/test/specs/MultiInput.spec.js

```diff
@@ -303,6 +303,32 @@ describe("MultiInput Truncated Token", () => {
 
 		assert.ok(await innerInput.getHTML(), "new MI should be displayed");
 	});
+
+	it("should open popover on keyboard combination ctrl + i", async () => {
+		const mi = await $("#truncated-token");
+		const token = await mi.$("ui5-token");
+		const rpoClassName = await getTokenizerPopoverId("truncated-token");
+		const rpo = await browser.$(`.${rpoClassName}`).shadow$("ui5-responsive-popover");
+
+		await mi.click();
+		await mi.keys(["Control", "i"]);
+		assert.ok(await rpo.getProperty("opened"), "Focused MI - More Popover should be opened");
```

Comment:
more popover?  It is n-more in general, but when in this case with only one token it is just a token popover, no n-more button is shown as it is only one long truncated token.

But it should be good to add a test covering the case with multiple tokens.

Reply:
done